build/contracts/Greeter.json

```diff
@@ -0,0 +1,2302 @@
+{
```

Comment:
Let’s put the vyper build folder in .gitignore

Reply:
agree, on it